Some README updates (#16)
inorbit-ai/ros_amr_interop: massrobotics_amr_sender_py/README.md

> Configuration-based ROS2 package for sending MassRobotics [AMR Interop Standard messages](https://github.com/MassRobotics-AMR/AMR_Interop_Standard) to complaint receivers.

Reviewer: ```suggestion
Configuration-based ROS2 package for sending MassRobotics [AMR Interop Standard messages](https://github.com/MassRobotics-AMR/AMR_Interop_Standard) to compliant receivers.
```

> # Installing

Reviewer: ```suggestion
# Package installation
```

## From binary packages

> The node is available as a released package and can be added manually to your ROS2 build installation:

Reviewer: ```suggestion
The node is available as a released package and can be added manually to your ROS2 build installation running the following command:
```

> ```sudo apt-get install ros-foxy-massrobotics-amr-sender```

Reviewer: ```suggestion
```console
$ sudo apt-get install ros-foxy-massrobotics-amr-sender
```

The above command installs the package via apt.
```

> Or listed as a rosdep dependency and installed with `rosdep update`

Reviewer: ```suggestion
Alternatively, the package can be listed as a rosdep dependency and installed with `rosdep update`
```

Reviewer: You could also change this to focus on the reader 

```suggestion
Alternatively, you can add the package as a rosdep dependency and then install it running `rosdep update`
```

## Building from source

> Make sure `ros2` is installed properly. Clone this repository inside your `src` folder on your local workspace and build the package:

Reviewer: ```suggestion
Make sure `ros2` is installed properly. Then clone this repository inside your `src` folder on your local workspace and build the package executing the following commands:
```

> ```bash

Reviewer: Does `bash` work on markdown ? I always used `console`

Author: it seems to work
